doc/Interns/KRMisha/MyFirstCommit.md

```diff
@@ -0,0 +1 @@
+2020-05-06
```

Comment:
Sorry, forgot to put it in the docs.  Can you add the time to this also?

Reply:
Done. Is the format okay?